lib/5.26.2/x86_64-linux/Config_heavy.pl

```diff
@@ -1038,15 +1038,15 @@ sub myconfig {
 inc_version_list=' '
 inc_version_list_init='0'
 incpath=''
-incpth='/usr/lib/gcc/x86_64-linux-gnu/4.7/include /usr/local/include /usr/lib/gcc/x86_64-linux-gnu/4.7/include-fixed /usr/include/x86_64-linux-gnu /usr/include'
+incpth='/usr/lib/gcc/x86_64-linux-gnu/6/include /usr/local/include /usr/lib/gcc/x86_64-linux-gnu/6/include-fixed /usr/include/x86_64-linux-gnu /usr/include'
 inews=''
 initialinstalllocation='/home/git/binary-com/perl/bin'
 installarchlib='/home/git/binary-com/perl/lib/5.26.2/x86_64-linux'
 installbin='/home/git/binary-com/perl/bin'
 installhtml1dir=''
 installhtml3dir=''
-installman1dir='/home/git/binary-com/perl/man/man1'
-installman3dir='/home/git/binary-com/perl/man/man3'
+installman1dir=''
```

Comment:
No man pages?

Reply:
Yep, we now have `-Dman1dir= -Dman3dir=` in our `./Configure` call.